### ISSUE TITLE
Options for androidVersionCode and iosBundleVersion

### ISSUE THREAD

**Issue by bennybennet** [SamVerschueren/gulp-cordova-version#1]
As this Plugin is using "cordova-config", i would love to see options for setting
- androidVersionCode
- iosBundleVersion

Bests,
Benny

**Comment by SamVerschueren** [comment]
Hi Benny

Good idea. Just thinking about creating a separate plugin or embedding it in this plugin. I think I go with embedding because it are version numbers. What do you prefer as implementation? Just passing the versionCode as second argument? 

``` javascript
gulp.task('build', function() {
    return gulp.src('dist')
        .pipe(create())
        .pipe(version('2.3.1', '2.300.1000'));
});
```

Or do developers like flexibility and have the possibility to use different version numbers for Android and iOS.

``` javascript
gulp.task('build', function() {
    return gulp.src('dist')
        .pipe(create())
        .pipe(version('2.3.1', { androidVersionCode: '2.300.1000', iosBundleVersion: '2.3.1'}));
});
```

The argument will be optional because not everyone wants to set it manually probably.

**Comment by bennybennet** [comment]
Hi Sam,

yeah, i would defenetly go for embedding it in this plugin.
I do like your second suggestion:

```
version('2.3.1', { androidVersionCode: '2.300.1000', iosBundleVersion: '2.3.1'})
```

Thanks for your fast response.

Benny

**Comment by SamVerschueren** [comment]
I can't give you an estimate on this because I have other priorities at the moment. Have to create new methods in [cordova-config](https://github.com/SamVerschueren/cordova-config) as well. I hope to add it in the beginning of next week. I'll keep you posted on this.

**Comment by bennybennet** [comment]
Thank you Sam.

Maybe this code could support:
https://github.com/shustariov-andrey/grunt-cordova-config/blob/master/tasks/cordova_config.js

**Comment by SamVerschueren** [comment]
Implemented in `0.2.0`.

**Comment by bennybennet** [comment]
Thanks :+1:
